www-apache/mod_wsgi/files/mod_wsgi-4.7.1-py310.patch

```diff
@@ -0,0 +1,125 @@
+From b439f1c411a9479ccc03c16465cdff50fede79d3 Mon Sep 17 00:00:00 2001
+From: Petr Viktorin <encukou@gmail.com>
+Date: Thu, 10 Jun 2021 15:45:03 +0200
+Subject: [PATCH] Use Py_CompileString rather than
```

Comment:
Where did this come from? Maybe put a link in there?

Reply:
It's simply backported from upstream https://github.com/GrahamDumpleton/mod_wsgi/commit/b439f1c411a9479ccc03c16465cdff50fede79d3
How would you like to mention it? Your suggestion is very welcome...

Reply:
Putting that link in the body of the patch email (that is, between the `Subject` header and the `---` preceding the diffstat), should work fine.